R/server.R

```diff
@@ -71,7 +71,7 @@ registerClient <- function(client) {
 #' @export
 addResourcePath <- function(prefix, directoryPath) {
   if (length(prefix) != 1) stop("prefix must be of length 1")
-  if (!grepl('^[a-z0-9\\-_][a-z0-9\\-_.]*$', prefix, ignore.case = TRUE, perl = TRUE)) {
+  if (!grepl('^[a-z0-9\\-_\\.][a-z0-9\\-_.]*$', prefix, ignore.case = TRUE, perl = TRUE)) {
```

Comment:
I think this can be simplified to just:
```
'^[a-z0-9\\-_.]+$'
```

However, we should also disallow prefixes that consist only of `.` characters - that could lead to unnecessary confusion.